src/components/Card/Card.tsx

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { FC, ReactChild } from "react";
+
+export type CardProps = {
+    children?: ReactChild;
+};
+
+export const Card: FC<CardProps> = ({ children }) => {
+    return (
+        <div data-test-id="card" className="tw-w-full tw-bg-white tw-border tw-border-black-10 tw-rounded">
+            {children}
+        </div>
```

Comment:
Could you ask Jamie about making a component?
We can't accept component that are not inside of the Figma Arcade file as components need to respect the Figma specs (aka the props used in Figma should be used in code)

Reply:
Hi @SamuelAlev 👋 

Thanks for the info!

Using [this task](https://app.clickup.com/t/1u470ae) as the input, and our current Brand Listening use case on [this prototype](https://www.figma.com/proto/F571BXcImisdo3ETFC3TSV/Social-Listening?node-id=88%3A104198&starting-point-node-id=286%3A192058), we have currently agreed with Jonas that this component won't have a header and divider.

But I will check with Jamie regarding getting it to the Figma Arcade file, thanks!

Reply:
Btw, Jamie has the Card designs in the WIP file here:
https://www.figma.com/file/pevuvey1joCsopTRr9f4OE/Arcade-Components?node-id=11443%3A363037